### PR TITLE
Increase main window size to 800x620, and unify the size for both mac, refactor MapPanel.java

### PR DESCRIPTION
**Pull request by DanVanAtta**
… and non-mac systems.

- Scrollbars are gone from the PBEM screen

Main screen Before:
before

After:
after

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/923)

### CODE REVIEW

File: src/games/strategy/engine/framework/startup/ui/MainPanel.java [hunk 1]
```diff
@@ -53,11 +53,7 @@
...
   static {
-    if (GameRunner.isMac()) {
-      m_initialSize = new Dimension(685, 620);
-    } else {
-      m_initialSize = new Dimension(625, 550);
-    }
+    m_initialSize = new Dimension(800, 620);
   }
 
   // private final Dimension m_initialSizeWithChat = new Dimension(500,650);
```
Comment (RoiEXLab) [inline src/games/strategy/engine/framework/startup/ui/MainPanel.java]: This comment should be removed

File: src/games/strategy/engine/framework/startup/ui/MainPanel.java [hunk 2]
```diff
@@ -53,11 +53,7 @@
   private static final Dimension m_initialSize;
 
   static {
-    if (GameRunner.isMac()) {
-      m_initialSize = new Dimension(685, 620);
-    } else {
-      m_initialSize = new Dimension(625, 550);
-    }
+    m_initialSize = new Dimension(800, 620);
```
Comment (RoiEXLab) [inline src/games/strategy/engine/framework/startup/ui/MainPanel.java]: This should be moved to the declaration of `m_initialSize`

**Comment by DanVanAtta** [comment]
Everything after the first commit is a refactor. Still plenty to do, but at least now 50 fewer lines of code.

**Comment by RoiEXLab** [comment]
@DanVanAtta no objections on merging in terms of code...
I just would like to know, why you chose 800x620 instead of common resolutions like 800x600

**Comment by DanVanAtta** [comment]
620 was the existing height on mac. The common resolution height is 700, so cannot be larger. I figure having some buffer so it is not full window is perhaps intended. So really then we are just widening the window. 800 is big enough that we don't scroll in PBEM, intention is to not be too drastic in widening the window - but to widen it enough.

**Comment by ron-murhammer** [comment]
@DanVanAtta Some issue with the ordering you are calling methods with the changes. I get the following error as 'playButton' can be null when 'setWidgetActivation()' is called:

Exception: null
java.lang.NullPointerException
	at games.strategy.engine.framework.startup.ui.MainPanel.setWidgetActivation(MainPanel.java:207)
	at games.strategy.engine.framework.startup.ui.MainPanel.<init>(MainPanel.java:57)
	at games.strategy.engine.framework.startup.ui.MainFrame.<init>(MainFrame.java:52)
	at games.strategy.engine.framework.GameRunner2$1.run(GameRunner2.java:157)

**Comment by DanVanAtta** [comment]
I did not test the refactor well enough, it got jacked somewhere along the way. I removed the last 5 commits from this PR, some bugs were introduced in those commits.

**Comment by DanVanAtta** [comment]
@ron-murhammer , note this is marked include with 1.9 release. The labels are simply intended to mean to review those PRs before the others. It does not mean to hold off on any other PRs - we probably will  wind up releasing stuff not labelled as 'include with 1.9 release' in the 1.9 release.

**Pull request merged**